e2e/test/scenarios/docker-compose.yml

```diff
@@ -2,6 +2,7 @@ name: metabase-e2e
 
 include:
   - ../../../snowplow/docker-compose.yml
+  - ../../../python-runner/compose.ci.yaml
```

Comment:
This will break when we no longer have a git submodule.

I suspect that @qnkhuat can give guidance on how to make this use a fresh shallow checkout of the repo instead.

Reply:
Should I merge this still and we can solve that once we move away from the submodule? Or should we move the `compose.ci.yaml` into this repo?

Reply:
Merging as-is makes the most sense - and I see you've already done so.

Bringing the compose file in is not enough, we'll need to get hold of the container as well. A problem for tomorrow.

Reply:
just clone it: https://github.com/metabase/metabase/pull/63654 :D